packages/automerge-repo-network-websocket/src/BrowserWebSocketClientAdapter.ts

```diff
@@ -1,8 +1,9 @@
 import {
   NetworkAdapter,
-  type PeerId,
```

Comment:
These type assertions allow imported type removal for transpilers without loading the full object graph. Is there a reason you took them out?